src/Maestrano.Net40.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Express 2013 for Windows Desktop
-VisualStudioVersion = 12.0.30501.0
+# Visual Studio 14
```

Comment:
Do you really need to change this ?
If not, we would prefer to keep your changes as minimal as possible